source/_components/lcn.markdown

```diff
@@ -10,11 +10,13 @@ footer: true
 logo: lcn.png
 ha_category:
   - Hub
+  - Cover
   - Light
   - Switch
 ha_release: 0.85
 ha_iot_class: "Local Push"
 redirect_from:
+  - /components/cover.lcn/
```

Comment:
We don't need a redirect from this page, since it never existed.

Reply:
Fixed. I got a little bit confused about how the docs should look like. Hope it‘s ok, now.